### PR TITLE
MAIN-354/355: Add client.now() for website and runner

### PR DESCRIPTION
**Pull request by sarahannali**

### CODE REVIEW

File: packages/client/index.js [hunk 1]
```diff
@@ -39,6 +39,11 @@ async function makeMove(move) {
   return parent.makeMove(move);
 }
 
+async function now() {
+  const parent = await connection.promise;
```
Comment (kevo1ution) [inline packages/client/index.js:43]: Is there a way to make this function asynchronous? That is the expectation from `semantle-battle perspective`

Comment (kevo1ution) [reply packages/client/index.js:43]: I'm thinking that we can default now to use the local Data.now() until the promise is ready

Comment (sarahannali) [reply packages/client/index.js]: done

Comment (sarahannali) [reply packages/client/index.js]: actually not done cant do bc penpal converts all to async anyway

**Review by kevo1ution — APPROVED** [review]

**Pull request merged**